src/components/Button/Button.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { RippleButton } from '../RippleButton';
+import type { ButtonProps } from './types';
+
+const Button: React.FC<ButtonProps> = ({
+  animationType,
+  style,
+  onPress,
+  animatedDuration = 500,
+  rippleColor = 'rgba(255, 255, 255, .25)',
+  children,
+}) => {
+  switch (animationType) {
```

Comment:
@nilesh-simform Why add a switch because in both cases use the same code?
CC. @mukesh-simform